oauthproxy.go

```diff
@@ -845,23 +852,23 @@ func (p *OAuthProxy) Authenticate(rw http.ResponseWriter, req *http.Request) int
 	}
 
 	if session != nil && session.IsExpired() {
-		log.Printf("%s removing session. token expired %s", remoteAddr, session)
+		logger.Printf("Removing session; token expired %s", session)
```

Comment:
Should this be a colon rather than semi-colon?
```suggestion
		logger.Printf("Removing session: token expired %s", session)
```

Reply:
I believe I was using semi-colons in the English sense of linking two related but different thoughts. Not quite sure why I used a mix of colon and semi-colon. I agree they should all colons - I've fixed all instances of this.